backend/src/main/java/gov/cdc/usds/simplereport/db/model/FeatureFlag.java

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+public class FeatureFlag extends AuditedEntity {
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private Boolean value;
+
+  public FeatureFlag() {
+    /* required for hibernate*/
+  }
```

Comment:
Can we remove this and just use the `@NoArgsConstructor`?

Reply:
+1, we probably also want to add an `@RequiredArgsConstructor` so that it's possible to add new feature flags outside of a manual database write.

Reply:
The @NoArgsConstructor was good but when I tried adding @RequiredArgsConstructor the project started to throw exceptions during build. Can this be added in the feature as need arises?